.github/workflows/nightly/README.md

```diff
@@ -0,0 +1,44 @@
+# Working on nightly
```

Comment:
Can you write a sentence or two, or a TL;DR, what this is about? What's the use case? Right now the first thing it says is "How to use dummify patches", but I don't know what a dummify patch is.